src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11687,36 +11687,36 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
 
             // Iteration_Ncomp: Perform iterations to calculate Ncomp (Label20)
             Counter = 1;
-        Label20:;
-            Ncomp_new = Ncomp;
-            Q_c_OU = max(0.0, Q_h_TU_PL - Ncomp);
-
-            // *VRF OU Te calculations
-            m_air = this->OUAirFlowRate * RhoAir;
-            SH_OU = this->SH;
-            this->VRFOU_TeTc(
-                state, HXOpMode::EvapMode, Q_c_OU, SH_OU, m_air, OutdoorDryBulb, OutdoorHumRat, OutdoorPressure, Tfs, this->EvaporatingTemp);
-            this->SH = SH_OU;
-
-            // *VRF OU Compressor Simulation at heating mode: Specify the compressor speed and power consumption
-            this->VRFOU_CalcCompH(state,
-                                  TU_HeatingLoad,
-                                  this->EvaporatingTemp,
-                                  Tdischarge,
-                                  h_IU_cond_out_ave,
-                                  this->IUCondensingTemp,
-                                  CapMinTe,
-                                  Tfs,
-                                  Pipe_Q_h,
-                                  Q_c_OU,
-                                  CompSpdActual,
-                                  Ncomp_new);
-
-            if ((std::abs(Ncomp_new - Ncomp) > (Tolerance * Ncomp)) && (Counter < 30)) {
+            bool not_converged;
+            do {
+                Ncomp_new = Ncomp;
+                Q_c_OU = max(0.0, Q_h_TU_PL - Ncomp);
+
+                // *VRF OU Te calculations
+                m_air = this->OUAirFlowRate * RhoAir;
+                SH_OU = this->SH;
+                this->VRFOU_TeTc(
+                    state, HXOpMode::EvapMode, Q_c_OU, SH_OU, m_air, OutdoorDryBulb, OutdoorHumRat, OutdoorPressure, Tfs, this->EvaporatingTemp);
+                this->SH = SH_OU;
+
+                // *VRF OU Compressor Simulation at heating mode: Specify the compressor speed and power consumption
+                this->VRFOU_CalcCompH(state,
+                                      TU_HeatingLoad,
+                                      this->EvaporatingTemp,
+                                      Tdischarge,
+                                      h_IU_cond_out_ave,
+                                      this->IUCondensingTemp,
+                                      CapMinTe,
+                                      Tfs,
+                                      Pipe_Q_h,
+                                      Q_c_OU,
+                                      CompSpdActual,
+                                      Ncomp_new);
+
+                bool not_converged = (std::abs(Ncomp_new - Ncomp) > (Tolerance * Ncomp));
```

Comment:
If you are still making changes in this PR, could I make the tiniest of requests?  Could you make the variable name `converged` instead of `not_converged`?  If we ever need to check if we are converged, we have to check `!not_converged`, and that's a more difficult mental parse, at least to me.  If you are done touching this code, then feel free to ignore, it's not all that important :)

Reply:
Thanks for the suggestion @Myoldmopar. I will change it to `converged` instead. I will continue working on this.